app-frontend/src/app/components/scenes/sceneImportModal/sceneImportModal.controller.js

```diff
@@ -664,7 +675,7 @@ export default class SceneImportModalController {
             .finally(() => {
                 this.isLoadingDatasources = false;
             }
-        );
+            );
```

Comment:
I disagree with your editor :rage: 
real talk - we should see if prettify will screw up with angular or if it will be nice like with the annotation app